### PR TITLE
Redesigned the model component.

### PR DESCRIPTION
**Pull request by goodguysarvagya**
This is my first PR, hope i don't break anything.

### CODE REVIEW

**Review by goodguysarvagya — COMMENTED** [review]
I don't know what is this probably prettier kicked in and did some fixing.

**Comment by goodguysarvagya** [comment]
@AndriyMulyar I have also redesigned some other components as result of those the lighthouse score went up by a lot feedback is appreciated. I will create further PRs with these fixes as per your guidance.

lighthouse_score_difference

**Comment by AndriyMulyar** [comment]
Really nice! Will review soon.

File: frontend/gpt4all.io/src/components/models.jsx [hunk 1]
```diff
@@ -1,88 +1,72 @@
...
-} from "./ui/select";
 
 const getModels = async () => {
-    const response = await fetch("https://gpt4all.io/models/models.json");
-    const jsonData = await response.json();
-    return jsonData;
-}
+  //const response = await fetch("https://gpt4all.io/models/models.json");
+  const response = await fetch(
+    "https://raw.githubusercontent.com/nomic-ai/gpt4all/main/gpt4all-chat/metadata/models.json"
```
Comment (AndriyMulyar) [inline frontend/gpt4all.io/src/components/models.jsx:8]: I want to retain this url: https://gpt4all.io/models/models.json

It redirects to the one you posted.

**Review by AndriyMulyar — CHANGES_REQUESTED** [review]

**Comment by AndriyMulyar** [comment]
Can you link a video of the new UI

**Comment by goodguysarvagya** [comment]
@AndriyMulyar I will upload the video asap. Thanks for the Quick PR Review. Much appreciated.

**Comment by goodguysarvagya** [comment]
> Can you link a video of the new UI

-  Here is a demo clip i changed the gifs to mp4 videos (smaller size) and rewrote the capability card component for that and performance table to have sorting although now i see i should place an indicator to show that the table is sortable.

> https://drive.google.com/file/d/18W1GDXcI4C4J9Lv7JJ4_6gu4Ux9PDXxY/view?usp=sharing

**Review by AndriyMulyar — APPROVED** [review]

**Pull request merged**